app/views/projects/blob/_actions.html.haml

```diff
@@ -17,6 +17,6 @@
         tree_join(@commit.sha, @path)), class: 'btn btn-sm'
 
 - if allowed_tree_edit?
-  = button_tag class: 'remove-blob btn btn-sm btn-remove',
-      'data-toggle' => 'modal', 'data-target' => '#modal-remove-blob' do
-    Remove
+  .btn-group{"aria-label" => "...", :role => "group"}
```

Comment:
Please remove the `aria-label` attribute, since "..." won't help screen readers at all.